uarch/compute-uarch-pristine-hash.cpp

```diff
@@ -113,11 +113,24 @@ int main(int argc, char *argv[]) try {
     proof_type proof = tree.get_proof(UARCH_STATE_START_ADDRESS, UARCH_STATE_LOG2_SIZE, nullptr);
     auto &uarch_state_hash = proof.get_target_hash();
 
+    // Print header
```

Comment:
Does this change remove the `xxd` requirement?

Reply:
No, this file generates only `uarch-pristine-hash.c`, but `uarch-pristine-ram.c` still needs `xxd` and this PR still includes your xxd fixes.